### DISCUSSION TITLE
assets plugin does not process scss files

### DISCUSSION THREAD

**Discussion by MaTsou** [jeremyevans/roda#359] [General]
I am using `roda 3.81.0`

There is a problem with sass-embedded gem but can't know what..

When `rackup` I got an error message :
```
ArgumentError: unknown keyword: :outvar
	/home/mazu/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/sass-embedded-1.77.5-x86_64-linux-gnu/lib/sass/compiler.rb:147:in `compile_string'
	/home/mazu/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/sass-embedded-1.77.5-x86_64-linux-gnu/lib/sass/embedded.rb:37:in `compile_string'
	/home/mazu/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/tilt-2.3.0/lib/tilt/sass.rb:19:in `_prepare_output'
```

My MWE :
```
# config.ru
require_relative 'app'
run App.freeze.app
```
```
# app.rb
require 'roda'
class App < Roda
  plugin :assets, css: 'app.scss'

  route do |r|
    r.assets

    "#{assets :css} hello world"
  end
end
```
```
# assets/css/app.scss
body { color: red; }
```
Generated html :
```
<html>
  <head>
    <link rel="stylesheet" href="/assets/css/app.scss">
  </head>
  <body>
    hello world
  </body>
</html>
```
Everything works fine if I replace .scss extension with .css
Please help ..

**Comment by jeremyevans** [comment]
Looks like a problem with tilt and not roda.  I maintain tilt as well, so I'll look into fixing the issue there.

**Reply by MaTsou** [reply]
Thanks.
Note that if I `plugin :render, escape: true` there is `unknown keyword: :escape` together with `:outvar`.

**Reply by jeremyevans** [reply]
Yes.  When sass-embedded is used, tilt's sass support doesn't ignore unsupported options (unlike when sass or sassc is used).  I'm working on fixing that.

**Reply by jeremyevans** [reply]
This issue should be fixed in tilt's master branch.  Can you try using that and confirm it fixes the problem?

**Reply by MaTsou** [reply]
Problem fixed.
Thank you very much for your reactivity. I discover Roda and found it great !!
